app/views/index.erb

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Welcome to Chitter.</title>
+	</head>
+	<body>
+		
+	</body>
```

Comment:
You don't need this stuff in every template, as the layout is always rendered first.